`npm install` fails: E401 Unauthorized

Repro:

On a clean environment:

- Download the repo
- `npm install`

Expected: Builds happy

Actual: 

```
npm ERR! code E401
npm ERR! 401 Unauthorized - GET https://npm.pkg.github.com/download/@seamapi/types/1.63.1/a78935453802a84e228b56bf39c0e632483d079f - authentication token not provided
```

--- a/package.json
+++ b/package.json
@@ -52,7 +52,7 @@
     "typescript": "^5.0.0"
   },
   "dependencies": {
-    "@seamapi/http": "^0.10.0",
+    "@seamapi/http": "^0.10.2",
     "ms": "^2.1.3",
     "redaxios": "^0.5.1",
     "swagger-parser": "^10.0.3"

